service-war/src/main/java/com/redshiftsoft/tesla/web/mvc/userroute/UserRouteController.java

```diff
@@ -64,9 +64,9 @@ public RouteSaveJsonResponse save(@RequestBody UserRouteDTO userRouteDto) {
 
     @PreAuthorize("isAuthenticated()")
     @Transactional
-    @RequestMapping(method = RequestMethod.GET, value = "/delete/{routeId}")
+    @RequestMapping(method = RequestMethod.POST, value = "/delete")
     @ResponseBody
-    public JsonResponse delete(@PathVariable Integer routeId) {
+    public JsonResponse delete(@RequestParam("id") Integer routeId) {
```

Comment:
If I had realized you were following the `{objecttype}Id` pattern for naming all the other params we're using for `delete` endpoints, I would've suggested we stick with `routeId`. :-) Let me know if you'd like to change it and I'll handle it on the front end.

Reply:
Haha if it was a real impact I would've mentioned it, but it's just 6 extra characters. All good!